etc/blob-indexer.conf-sample

```diff
@@ -1,16 +1,33 @@
 [blob-indexer]
+
+# Namespace name
 namespace = NS
+
+# Run daemon as user
 user = openio
+
+# Logging configuration
+log_level = INFO
+log_facility = LOG_LOCAL0
+log_address = /dev/log
+syslog_prefix = OIO,OPENIO,blob-index,1
```

Comment:
If the namespace is "NS", the syslog prefix should be "OIO,OPENIO,blob-index,1".